articles/azure/azs-how-link-storage-explorer.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How to link Azure Storage Explorer to Azure Stack | UKCloud Ltd
+description: Provides help for linking Azure Storage Explorer to Azure Stack
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to link Azure Storage Explorer to Azure Stack
+toc_fullpath: Users/How To/Create a virtual machine/azs-how-link-storage-explorer.md
+toc_mdlink: azs-how-link-storage-explorer.md
+---
+
+# How to link Azure Storage Explorer to Azure Stack
+
+In this guide, you will learn how to link Azure Stack subscriptions to the Azure storage explorer. Azure storage explorer is a standalone application that allows you to easily work with Azure Stack storage data on Windows, macOS and Linux. This tool allows for easy downloading and uploading to Azure Stack which makes tasks such as downloading and uploading data disks to VM's very easy.
+
+If you need to install the storage explorer, please find the link [here](https://azure.microsoft.com/en-us/features/storage-explorer/) and install it.
+
+## Installation and setup
+
+1. Download and install storage explorer.
+
+    ![Storage tool installation guide 1](images/azs-browser-storage-tool-download.png)
+
+2. Once launched navigate to accounts.
+
+     ![Storage tool installation guide 2](images/azs-installation-storage-tool-1.png)
+
+3. Select **Add an account...**
+
+    ![Storage tool installation guide 3](images/azs-installation-storage-tool-2.png)
+
+4. Select **Add an Azure Account** > **Use Custom Environment**.
+
+    ![Storage tool installation guide 4](images/azs-installation-storage-tool-3.png)
+
+5. Name the enviroment **Azure Stack** and for the ARM endpoint use **[https://management.frn00006.azure.ukcloud.com]()**.
```

Comment:
I don't think the URL here needs to be a link, so just enclose it with backticks `https://management.frn00006.azure.ukcloud.com`

Reply:
fixed